bot/exts/filters/webhook_remover.py

```diff
@@ -13,8 +13,8 @@
 
 ALERT_MESSAGE_TEMPLATE = (
     "{user}, looks like you posted a Discord webhook URL. Therefore, your "
-    "message has been removed. Your webhook may have been **compromised** so "
-    "please re-create the webhook **immediately**. If you believe this was a "
+    "message has been removed, alongside with your webhook"
+    "you can re-create it if you wish to. If you believe this was a "
     "mistake, please let us know."
```

Comment:
This says
> Bluenix, looks like you posted a Discord webhook URL. Therefore, your message has been removed, alongside with your webhook you can re-create it if you wish to. If you believe this was a mistake, please let us know.

You should add a period between "your webhook" and "you can re-create it if you wish to".